README.md

```diff
@@ -143,8 +194,24 @@ Your query here
 
 8. Show the uuid of the employee, first_name and lastname combined, email, job_title, the name of the office they belong to, the name of the country, the name of the state and the name of the boss (boss_name)
 
-```
-Your query here
+```sql
+SELECT  e1.uuid,
+        CONCAT(e1.first_name,' ',e1.last_name) as full_name,
+        e1.email,
+        e1.job_title,
+        o.name as company,
+        c.name as country,
+        s.name as state,
+        e2.first_name as boss_name
```

Comment:
for more readability instead of `e1` and `e2` you can use e.g. `employee` and `boss`